app/assets/stylesheets/home.scss

```diff
@@ -1,6 +1,6 @@
 .masthead-video {
   background-color: black;
-  position: fixed;
+  position: absolute;
   top: 50%;
```

Comment:
Can it be fixed in a separate PR? It is likely a margin issue or a result of the parent's' width settings.